c2corg_ui/static/js/documentediting.js

```diff
@@ -33,7 +33,14 @@ app.documentEditingDirective = function() {
     scope: true,
     controller: 'appDocumentEditingController',
     controllerAs: 'editCtrl',
-    bindToController: true
+    bindToController: true,
+    link: function(scope, el, attrs, ctrl) {
+      // The directive executes after the dom is loaded but we have to wait until the expressions are evaluated
```

Comment:
in my case yes, but maybe it's thanks to a fast computer and internet
connection. :D
Trying to do that with constants.

On Fri, Mar 4, 2016 at 1:33 PM, Tobias Sauerwein <notifications@github.com>
wrote:

> In c2corg_ui/static/js/documentediting.js
> <https://github.com/c2corg/v6_ui/pull/220#discussion_r55024386>:
>
> > @@ -33,7 +33,14 @@ app.documentEditingDirective = function() {
> >      scope: true,
> >      controller: 'appDocumentEditingController',
> >      controllerAs: 'editCtrl',
> > -    bindToController: true
> > +    bindToController: true,
> > +    link: function(scope, el, attrs, ctrl) {
> > +      // The directive executes after the dom is loaded but we have to wait until the expressions are evaluated
>
> The problem is not that it will appear 0.5 seconds later. The problem is
> that you have no guarantee that 500ms are are always long enough. Are the
> elements that you want to count always ready at that moment?
>
> It would be good if you could avoid this timeout.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/c2corg/v6_ui/pull/220/files#r55024386>.
>



-- 

*Adam Gina*JavaScript Developer
adam.gina@camptocamp.com

*Camptocamp SA*EPFL Innovation Park
Bâtiment A
1015 Lausanne

Tel + 41 21 619 10 10
http://www.camptocamp.com